docs/references/keywords.rst

```diff
@@ -183,8 +183,12 @@ extensions).
 
 ``data_files``
     .. warning::
-        ``data_files`` is deprecated. It does not work with wheels, so it
-        should be avoided.
+        This is an advanced feature and it is *not intended to work with absolute installation paths*.
+        All files listed in ``data_files`` will be installed in paths relative to a directory
+        decided by the package installer (e.g. `pip`),
+        which usually results in nesting under a virtual environment.
+        See :doc:`userguide/datafiles` for an alternative placing inside the package directory.
```

Comment:
now this is raising /home/runner/work/setuptools/setuptools/docs/references/keywords.rst:186: WARNING: unknown document: 'userguide/datafiles'

Reply:
```suggestion
        See :doc:`/userguide/datafiles` for an alternative placing inside the package directory.
```

Reply:
Thank you for spotting that and for the changes!